cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -85,6 +73,14 @@ def assign_chargeback_rate(new_compute_rate):
     enterprise.storageassign()
 
 
+def verify_vm_uptime(appliance, provider, vmname):
+    """Verify VM uptime is at least one hour.That is the shortest duration for
+    which VMs can be charged.
+    """
```

Comment:
REQUIRED: please use summary line followed by blank line followed by more detailed description to be consistent with our [docstrings style](https://gist.github.com/mfalesni/11694e0d563c80c3917ce6012400b93f).
I.e.
```python
def verify_vm_uptime(appliance, provider, vmname):
    """Verifies that VM uptime is at least one hour.

    One hour is the shortest duration for which VMs can be charged.
    """
```

Reply:
done